services/datasources/spec/acceptance/datasources_factory_spec.rb

```diff
@@ -11,7 +11,7 @@
 describe DatasourcesFactory do
 
   def get_config
-    @config ||= YAML.load_file("#{File.dirname(__FILE__)}/../../../../config/app_config.yml")['defaults']
```

Comment:
holy smokes! Could you Pr this to carto as well?

Reply:
Done. https://github.com/CartoDB/cartodb/issues/13272